README.md

```diff
@@ -1,18 +1,63 @@
 Apache SkyWalking CLI
 ===============
 
+![](https://github.com/apache/skywalking-cli/workflows/Build/badge.svg?branch=master)
+![](https://codecov.io/gh/apache/skywalking-cli/branch/master/graph/badge.svg)
+
 <img src="http://skywalking.apache.org/assets/logo.svg" alt="Sky Walking logo" height="90px" align="right" />
 
 The CLI (Command Line Interface) for [Apache SkyWalking](https://github.com/apache/skywalking).
 
 SkyWalking CLI is a command interaction tool for the SkyWalking user or OPS team, as an alternative besides using browser GUI.
 It is based on SkyWalking [GraphQL query protocol](https://github.com/apache/skywalking-query-protocol), same as GUI.
 
+# Install
+As SkyWalking CLI is using `Makefile`, compiling the project is as easy as executing a command in the root directory of the project.
+
+```shell
+git clone https://github.com/apache/skywalking-cli
+cd skywalking-cli
+make clean && make
+```
+
+and copy the `./bin/swctl` to your `PATH` directory, usually `/usr/bin/` or `/usr/local/bin`, or you can copy it to any directory you like,
+and add that directory to `PATH`.
+
 # Commands
-TODO
+Commands in SkyWalking CLI are organized into two levels, in the form of `swctl --option <level1> --option <level2> --option`,
+there're options in each level, which should follow right after the corresponding command, take the following command as example:
+
+```shell
+$ swctl --debug service list --start="2019-11-11" --end="2019-11-12"
+```
+
+where `--debug` is is an option of `swctl`, and since the `swctl` is a top-level command, `--debug` is also called global option,
+and `--start` is an option of the third level command `list`, there is no option for the second level command `service`.
```

Comment:
The description is not consistent, should we call second or third level command?